src/Sfx/App/Scripts/Services/RestClientService.ts

```diff
@@ -571,6 +571,17 @@ module Sfx {
             return this.getEvents(FabricEvent, url, null, null, messageHandler);
         }
 
+        public restartReplica(nodeName: string, partitionId: string, replicaId: string, messageHandler?: IResponseMessageHandler): angular.IPromise<{}> {
+            let url = "Nodes/"

```

Comment:
I like following format more because it is easy to get the Url structure 
`/nodes/${nodeName}/$/GetPartitions/${partitionId}/$/GetReplicas/${replicaId}/$/Restart`

Reply:
Fixed to use that.